src/hooks/resourceHooks/useLocalGokbPkg.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { PACKAGES_ENDPOINT } from '../../constants';
+
+const useLocalGokbPkg = ({ reference }) => {
+  const ky = useOkapiKy();
+  const remoteId = reference ? reference.split(':')[0] : null;
+  const { data } = useQuery(
+    ['PKG', 'fetchLocalPkg', remoteId],
```

Comment:
This isn't right.


I believe we already have a usePackage? If so we should be trying to work out how to wrap that, not create an entirely new one.

Our usual pattern here is ['ERM', 'ObjectName', 'id-if-relevant', ...otherSpecifiers, ...queryStuffs]